backend/resources/akvo/lumen/migrations/tenants/010-invites.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE invite (
+     id text PRIMARY KEY DEFAULT gen_random_uuid()::text,
+     email text NOT NULL,
+     expiration_time timestamptz,
+     consumed_at timestamptz,
```

Comment:
I was thinking about the naming here with `expiration_time` vs. `consumed_at`? Is there an inconsistency? Perhaps `expiration_at`? If these differ in naming I could be fooled to believing that expiration_time would be an integer (in `ms`, `s` or something)? We also haven't added postfix (either `_at` or `_time`) to other `timestampz`'s.

I was also wondering what `consumed_at` means? I suppose it means time when the invitation was accepted? So we don't track if it was accepted/rejected/canceled? Only the time when one of these happened? (not that we currently can reject/cancel invitation)